dear_petition/petition/tests/factories.py

```diff
@@ -60,13 +61,6 @@ def record_data(idx):
                     "Law": "G.S. 20-141(B)",
                     "Code": "4450",
                 },
-                {
-                    "Action": "CONVICTED",
-                    "Description": "IMPROPER EQUIP - SPEEDOMETER",
-                    "Severity": "INFRACTION",
-                    "Law": "G.S. 20-123.2",
-                    "Code": "4418",
-                },
```

Comment:
FYI I don't think this test fixture is actually used. You can delete the offenses from this block and it won't make any tests fail